visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1248,10 +1248,17 @@ void MotionPlanningDisplay::load(const rviz::Config& config)
   if (frame_)
   {
     QString host;
-    if (config.mapGetString("MoveIt_Warehouse_Host", &host))
+    ros::NodeHandle nh;
+    std::string host_param;
+    if (nh.getParam("warehouse_host", host_param))
+    {
+      host = QString::fromStdString(host_param);
+      frame_->ui_->database_host->setText(host);
+    }
+    else if (config.mapGetString("MoveIt_Warehouse_Host", &host))
       frame_->ui_->database_host->setText(host);
     int port;
-    if (config.mapGetInt("MoveIt_Warehouse_Port", &port))
+    if (nh.getParam("warehouse_port", port) || config.mapGetInt("MoveIt_Warehouse_Port", &port))
```

Comment:
this appears to be a breaking change in functionality in that the way settings are loaded are different. i think this PR would need to target ROS Kinetic branch